godtools/App/Features/ToolSettings/Presentation/ToolSettings/ToolSettingsHostingView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ToolSettingsHostingView.swift
+//  godtools
+//
+//  Created by Levi Eggert on 5/12/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+class ToolSettingsHostingView: UIHostingController<ToolSettingsView> {
+    
+    private let modalHeightPercentageOfScreen: CGFloat = 0.6
+    private let modalHorizontalPadding: CGFloat = 12
+    private let modalCornerRadius: CGFloat = 12
+    
+    private var modalBottomToParent: NSLayoutConstraint?
+    
+    required init(view: ToolSettingsView) {
+        
+        super.init(rootView: view)
+        
+        setupLayout()
+    }
+    
+    @MainActor required dynamic init?(coder aDecoder: NSCoder) {
```

Comment:
Haven't seen this `@MainActor` thing before, looks cool!  Guess we'll need to use Combine before we can actually make use of it though.